deployment/ansible/deploy.yml

```diff
@@ -40,7 +40,7 @@
           port: 3013
       chain:
         persist: true
-        db_path: "./db30"
+        db_path: "./db31"
```

Comment:
Is the consensus affected?

Reply:
Yes, some tx types has changed name so their serialization will be different

Reply:
So as things stand now we shall bump block and genesis version. Though it can be done in separate PR.

Reply:
Tracked as https://www.pivotaltracker.com/story/show/155400352 so we remember.